Content.Server/Light/EntitySystems/ToggleableOccluderSystem.cs

```diff
@@ -0,0 +1,58 @@
+using Content.Server.DeviceLinking.Events;
+using Content.Server.DeviceLinking.Systems;
+using Content.Server.Light.Components;
+using Robust.Shared.GameObjects;
+
+namespace Content.Server.Light.EntitySystems
```

Comment:
```suggestion
namespace Content.Server.Light.EntitySystems;
```
Use a file-scoped namespace and remove the curly braces below